src/runtime/pkg/containerd-shim-v2/shim_management.go

```diff
@@ -312,7 +312,46 @@ func GetSandboxesStoragePathRust() string {
 	return "/run/kata"
 }
 
-// ServerSocketAddress returnes the address of the unix domain socket the shim management endpoint
+-// ServerSocketAddress returnes the address of the unix domain socket the shim management endpoint
```

Comment:
Hmm... something went wrong. All lines now start with `+` sign...

Reply:
Fixed, should be ok now.